{{ cookiecutter.package_name }}/docs/conf.py

```diff
@@ -33,7 +33,8 @@
 try:
     from sphinx_astropy.conf.v1 import *  # noqa
 except ImportError:
-    print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+    print('ERROR: the documentation requires the sphinx-astropy package '
+          'to be installed')
```

Comment:
do we go with 80 wide lines here, or have switched to 100 project wise? (I don't know, just asking)

Reply:
Errr... My Emacs was complaining as there is no `flake8` setting in the package level `setup.cfg`.